### DISCUSSION TITLE
Simulator + Arcade Mode

### DISCUSSION THREAD

**Discussion by Erisbv-zz** [JSBSim-Team/jsbsim#622] [Q&A]
Hello, peace and light to all members.

I would like to know if it is possible to develop a simple simulator, almost an arcade mode? Example; equipped/unequipped jet weight, speed, among others.

Thanks in advance for any help, and if possible, I'm looking for a complete tutorial in Python or Unreal, preferably Unreal

**Comment by seanmcleod** [comment]
I'm not aware of any complete tutorials. Have you looked at the documentation and source code for the Unreal reference app? 

https://github.com/JSBSim-Team/jsbsim/tree/master/UnrealEngine

**Comment by ICanDoThings2** [comment]
If I understand what you're asking, I don't think JSBSim really supports changing runtime variables much, like engine speed and so on.

**Reply by Erisbv-zz** [reply]
> If I understand what you're asking, I don't think JSBSim really supports changing runtime variables much, like engine speed and so on.

Yes that, I would like to slow down and possibly nullify the G-effect (AOA), limit altitude, increase the flap force to execute shorter maneuvers...

**Comment by seanmcleod** [comment]
@Erisbv you'll need to be a bit more specific/detailed on what you mean by "nullify the G-effect", "limit altitude" etc.

> increase the flap force

You mean increase the amount of extra lift that flaps would generate? What about the extra drag? Why can't you edit the aircraft xml file to change those?

**Reply by Erisbv-zz** [reply]
> @Erisbv you'll need to be a bit more specific/detailed on what you mean by "nullify the G-effect", "limit altitude" etc.
> 
> > increase the flap force
> 
> You mean increase the amount of extra lift that flaps would generate? What about the extra drag? Why can't you edit the aircraft xml file to change those?

forgive me for expressing myself in a limited way, I don't speak fluent English.

In the jet simulation, the performance is limited to reality, that is: blackout (fainting) due to the G force, long curves due to sustainability, stall...

What I'm looking for is to know if this library has the option to change this data, bringing it closer to the arcade mode, however, without losing the realism of the physics effects in the game.

**Comment by seanmcleod** [comment]
> performance is limited to reality

Performance is based on the data that is supplied for the aircraft model. So if you want, you could edit the aircraft's xml file for example and make it's drag 0 or close to 0 to change it's performance, change the slope of the lift curve etc.

So first thing to do is look at how the aerodynamic and engine data is supplied and familiarise yourself with them. You'll then see what sorts of changes you could make to the data to get the aircraft to perform as you want.

**Reply by Erisbv-zz** [reply]
> > performance is limited to reality
> 
> Performance is based on the data that is supplied for the aircraft model. So if you want, you could edit the aircraft's xml file for example and make it's drag 0 or close to 0 to change it's performance, change the slope of the lift curve etc.
> 
> So first thing to do is look at how the aerodynamic and engine data is supplied and familiarise yourself with them. You'll then see what sorts of changes you could make to the data to get the aircraft to perform as you want.

Thank you very much, having this as confirmation, I will start the detailed studies.

Gratitude
peace and light
